radicle-cli/src/commands/remote.rs

```diff
@@ -0,0 +1,126 @@
+//! Remote Command implementation
+#[path = "remote/add.rs"]
+pub mod add;
+#[path = "remote/list.rs"]
+pub mod list;
+#[path = "remote/rm.rs"]
+pub mod rm;
+
+use std::ffi::OsString;
+
+use anyhow::anyhow;
+
+use radicle::prelude::Did;
+
+use crate::terminal as term;
+use crate::terminal::args::{string, Error};
+use crate::terminal::{Args, Context, Help};
+
+pub const HELP: Help = Help {
+    name: "remote",
+    description: "Manage set of tracked repositories",
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+    rad remote
+    rad remote list
+    rad remote add <url> [--alias]
+    rad remote rm <alias>
+Options
+        --help                 Print help
+"#,
+};
+
+#[derive(Debug, Default, PartialEq, Eq)]
+pub enum OperationName {
+    Add,
+    Rm,
+    #[default]
+    List,
+}
+
+#[derive(Debug)]
+pub enum Operation {
+    Add { did: Did, alias: Option<String> },
+    Rm { alias: String },
+    List,
+}
+
+#[derive(Debug)]
+pub struct Options {
+    pub op: Operation,
+    pub verbose: bool,
+}
+
+impl Args for Options {
+    fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)> {
+        use lexopt::prelude::*;
+
+        let mut parser = lexopt::Parser::from_args(args);
+        let mut op: Option<OperationName> = None;
+        let mut did: Option<Did> = None;
+        let mut alias: Option<String> = None;
+        let mut verbose = false;
+
+        while let Some(arg) = parser.next()? {
+            match arg {
+                Long("help") => {
+                    return Err(Error::Help.into());
+                }
+                Long("verbose") | Short('v') => {
+                    verbose = true;
+                }
+                Long("alias") | Short('a') => {
+                    let value = parser.value()?;
+                    let value = string(&value);
```

Comment:
we typically do `parser.value()?.to_string_lossy().into()`

Reply:
`string` does exactly what I introduced in my previous PR so I guess now this is the easy way to handle the OsString?